nameable.rb

```diff
@@ -0,0 +1,30 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [Optional] Please let's raise the error like this:
`raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`

Reply:
Thanks a lot for the suggestion. I will implement it in the next pull request.